README.md

```diff
@@ -68,7 +68,10 @@ The following 62 microcontroller boards are currently supported:
 * [Arduino Mega 2560](https://store.arduino.cc/arduino-mega-2560-rev3)
 * [Arduino MKR1000](https://store.arduino.cc/arduino-mkr1000-wifi)
 * [Arduino Nano](https://store.arduino.cc/arduino-nano)
-* [Arduino Nano33 IoT](https://store.arduino.cc/nano-33-iot)
+* [Arduino Nano 33 BLE](https://store.arduino.cc/nano-33-ble)
+* [Arduino Nano 33 BLE Sense](https://store.arduino.cc/nano-33-ble-sense)
```

Comment:
`targets/nano-33-ble-sense.json` does not exist.
Is this configuration correct?
If there is even a slight difference in pin placement, I think I need to add `board_nano-33-ble-sense.go` or something similar.

Reply:
Yes, there is bit of mess there.
Silkscreen pins are exactly the same.
It's just Sense variant has more modules on board that can be accessed via I2C or SPI.

Compare:
https://content.arduino.cc/assets/Pinout-NANOble_latest.pdf
and
https://content.arduino.cc/assets/Pinout-NANOsense_latest.pdf

Reply:
It looks like nano-33-ble-sense is just nano-33-ble with some additional part implementations.
If this is the case, `targets/nano-33-ble-sense` may not be necessary.

The difference between the board name in README.md and the name specified by -target is getting more and more difficult.

Reply:
I think it would be easier for tinygo users to understand if there was a targets/nano-33-ble-sense.json.
See targets/clue.json and targets/clue-alpha.json.

https://github.com/tinygo-org/tinygo/blob/release/targets/clue.json
https://github.com/tinygo-org/tinygo/blob/release/targets/clue-alpha.json

Do you have any thoughts on this? @deadprogram

Reply:
Can we perhaps have `targets/nano-33-ble-sense.json` with only content as following?
```
{"inherits":["nano-33-ble"]}
```
This basically implements an alias.

Reply:
> Can we perhaps have targets/nano-33-ble-sense.json with only content as following?

That seems to me to be the most consistent approach.